### PR TITLE
fix: [filedialog] the recent item not hide

### PR DESCRIPTION
**Pull request by GongHeng2017**
cause:
not judge the recent item from dconfig.
solution:
add the judge by dconfig.

Log: fix issue
Bug: https://pms.uniontech.com/bug-view-238675.html

### CODE REVIEW

File: src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp [hunk 1]
```diff
@@ -423,7 +424,11 @@ void FileDialog::setAcceptMode(QFileDialog::AcceptMode mode)
     if (mode == QFileDialog::AcceptOpen) {
         statusBar()->setMode(FileDialogStatusBar::kOpen);
         setFileMode(d->fileMode);
-        urlSchemeEnable("recent", true);
+        QVariantMap vMap = DConfigManager::instance()->value("org.deepin.dde.file-manager.sidebar", "itemVisiable").toMap();
+        if (vMap.contains("recent")) {
```
Comment (itsXuSt) [inline src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp]: vMap.value("recent", true).toBool()
你写的太冗杂了。上面这行代码在组策略中没有的情况下会反悔默认值 true。

Comment (itsXuSt) [reply src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp]: URLSchemeEnable("recent", vMap.value("recent", true).toBool())

Comment (GongHeng2017) [reply src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp]: done

**Review by itsXuSt — APPROVED** [review]

**Review by max-lvs — APPROVED** [review]

**Pull request merged**